docs/whats-new.md

```diff
@@ -12,6 +12,10 @@ Target Release Date: mid-Feb, 2019. `alpha` releases will be made available.
 
 - `ImageBitmap` Loading: `load(..., ImageLoader, {image: {type 'imagebitmap'}}) => ImageBitmap`
 
+**@loaders.gl/json**
+
+- The streaming JSON loader now has an experimental option `_container` that returns the JSON object containing the JSON array being streamed (as additional first and last batches).
+
```

Comment:
`_container` is not quite intuitive for what it does.

Reply:
> _container is not quite intuitive for what it does.

Agreed. Nothing short and clear comes to mind... Changed to `_rootObjectBatches` for now.